internal/distro/rhel85/partition_tables.go

```diff
@@ -143,6 +143,94 @@ func defaultPartitionTable(imageOptions distro.ImageOptions, arch distro.Arch, r
 	panic("unknown arch: " + arch.Name())
 }
 
+func ec2PartitionTable(imageOptions distro.ImageOptions, arch distro.Arch, rng *rand.Rand) disk.PartitionTable {
+	var sectorSize uint64 = 512
+	if arch.Name() == "x86_64" {
+		return disk.PartitionTable{
+			Size: imageOptions.Size,
+			UUID: "D209C89E-EA5E-4FBD-B161-B461CCE297E0",
+			Type: "gpt",
```

Comment:
is it really `gpt`, non-uefi partition table? :O Not an expert, isn't that uncommon?

Reply:
I must admit that I'm no expert either. I basically used values from image-info report for current RHEL ec2 images:
```json
  "partition-table": "gpt",
  "partition-table-id": "A03D2C64-F209-407F-B699-3E8519C9771E",
  "partitions": [
    {
      "bootable": false,
      "fstype": null,
      "label": null,
      "partuuid": "21D40DD5-AC59-41B9-A1A8-21C9FE02C3CB",
      "size": 1048576,
      "start": 1048576,
      "type": "21686148-6449-6E6F-744E-656564454649",
      "uuid": null
    },
    {
      "bootable": false,
      "fstype": "xfs",
      "label": null,
      "partuuid": "E6934A8A-B0C2-4FC3-8170-E872167C3446",
      "size": 6442450944,
      "start": 2097152,
      "type": "0FC63DAF-8483-4772-8E79-3D69D8477DE4",
      "uuid": "6c1f2123-a6e2-4b97-8511-47913c094f88"
    }
  ],
```

I'm happy to investigate this further if you have any pointers or ideas what to look for.

Reply:
The partitioning is the same as on the official RHEL images and the image boots successfully. I'm marking this comment as resolved.